core/src/test/java/org/veriblock/sdk/ServiceSecurityPayloadTests.java

```diff
@@ -0,0 +1,112 @@
15625BB8C044B3173EBCC7CFD8B346DE6C59A448942D97C56A068B36386CEA4CDBA0F2D1AD5DE5499C0405F5E10FE7F2724000019CDF0002E195A99911568405BB38FA44B8C044B3173EBCC7CFD8B346DE6C59A448948561B42A78E9BCC33A33234A7CD963005DE54A2D0405F5E149DBEC2F4000019CE000028B4D485146F4C81E6C5A34BDB8C044B3173EBCC7CFD8B346DE6C59A44894BBB1A2EDB7F26931492C81D5BCBF297C5DE54A460405F5E143EA5E494000019CE10002418B53CD808D1DCCCDCDC18CB8C044B3173EBCC7CFD8B346DE6C59A44894D20E8FD4148C721A75D6A61610A7D7A15DE54AB80405F5E14D1C5ADD4000019CE20002793B8AB2DCF53F5327869F41B8C044B3173EBCC7CFD8B346DE6C59A4489468F160DD48B72FAAB107F3A111CDD8125DE54B030405F5E154F077934000019CE300023CBAA41FF8F9C075EEC94FAAB8C044B3173EBCC7CFD8B346DE6C59A448941620F988476A36813FB17DD975101E1F5DE54B230405F5E15A0298144000019CE40002B5AB985E1B46440F0CEC57AAB8C044B3173EBCC7CFD8B346DE6C59A448943A05A7A699ED7356AE9E240E7D7B0D815DE54B460405F5E15B36DE424000019CE5000262AA23453C80EBA154978C13B8C044B3173EBCC7CFD8B346DE6C59A448949150A3C3E6313C2E47B36C4FA9E617875DE54BCF0405F5E164F538974000019CE60002ACC08C934DCDE66730B919F6453C80EBA154978C13B8C044B3173EBCC7CF16142FE0180AA77E08B52483D8E5879B5DE54C4E0405F5E141F005364000019CE7000285ADBE6E86A97DB0B2329A52453C80EBA154978C13B8C044B3173EBCC7CFF8E4064EDA8B11B6BCCC97686A7EBCD15DE54C4F0405F5E17170B8BD4000019CE8000263AF0957BABBB5DBC010963A453C80EBA154978C13B8C044B3173EBCC7CF71EEBA40FE79E3CE8B5E064D32238A705DE54C510405F5E1717EB34F4000019CE90002486750A9CAD9BB94F7FAFFE4453C80EBA154978C13B8C044B3173EBCC7CF5BD5A8779F1F28CEB857D05C8DED843D5DE54CCF0405F5E159A9F3FF4000019CEA0002372484B65E4D76004BE70F59453C80EBA154978C13B8C044B3173EBCC7CF961EBDAF088E1E4B68D905089BDE81F55DE54CD60405F5E17ED14C004000019CEB000268EE42008FE514406BD61D5A453C80EBA154978C13B8C044B3173EBCC7CF5F226C7B6FDC16DE014D2E5C922295DA5DE54D550405F5E105924A214000019CEC000201DDD3BBCAD76B681BC2A1E8453C80EBA154978C13B8C044B3173EBCC7CF27C871A3EF32596E74CF1718893EE6A65DE54D7E0405F5E10B3020194000019CED00020F344EB4782ACECF6EF1AB67453C80EBA154978C13B8C044B3173EBCC7CFADFAA159D73F0128BEC9A0BA09BCC67F5DE54D990405F5E10F18FA024000019CEE0002031A7EE5B9CDCF040DCEC2EF453C80EBA154978C13B8C044B3173EBCC7CFA9CB475021388EBB8F304F33852BBB285DE54DBC0405F5E11026120A4000019CEF000200C0161E988A0F30ACEBDE0B453C80EBA154978C13B8C044B3173EBCC7CF320F4D720587FC39A299D00476B078CD5DE54EB30405F5E124F064604000019CF00002B0E52A13C61B5D2BC95030AF453C80EBA154978C13B8C044B3173EBCC7CF5D83E7B7D30DA514FA580D91563E591B5DE54ECB0405F5E1537CE4A94000019CF100023C7C7D26528E6A9E75947316453C80EBA154978C13B8C044B3173EBCC7CFBCAA574FCCA4DDECF22928931E98E4B55DE54EE80405F5E1276AB8854000019CF200021F059032E4E3DB3C9542859F453C80EBA154978C13B8C044B3173EBCC7CF0FAD624290F7D0883AC85DD417CB89DA5DE54F380405F5E1388BF5004000019CF30002342586773CC168F894BB8630453C80EBA154978C13B8C044B3173EBCC7CF6C9A115D7355B6F38A014585A9633A9C5DE54F9E0405F5E13767E2804000019CF40002C8640A0753B18CB2AD86F746453C80EBA154978C13B8C044B3173EBCC7CF89AA14DBF6D21C2CD6CE8F8AD85DD45F5DE54FCC0405F5E13CD6E2744000019CF50002DBBD55AE2BB18CB5217BE6E5453C
```

Comment:
will test pass if you enable difficulty validation?

Reply:
no